wasp.go

```diff
@@ -65,10 +67,17 @@ type CallResult struct {
 }
 
 const (
-	RPSScheduleType string = "rps_schedule"
-	VUScheduleType  string = "vu_schedule"
+	RPSScheduleType    string = "rps_schedule"
+	CustomScheduleType string = "custom_schedule"
```

Comment:
It'd be great if we can omit this type and use a default value of RateLimitUnitDuration = 1 sec, if it's not 1 sec then we set your value. Works only for RPSScheduleType, anyway.